pywbem_mock/_providerdispatcher.py

```diff
@@ -213,6 +213,14 @@ def CreateInstance(self, namespace, NewInstance):
                         "exist in namespace {1!A} of the CIM repository.",
                         NewInstance.classname, namespace))
 
+        if "Abstract" in creation_class.qualifiers:
+            raise CIMError(
+                CIM_ERR_FAILED,
+                _format("Abstract class cannot be instantiated. Creation class "
+                        "{0!A} of new instance in Namespace {1!A} "
+                        "of the CIM repository has Abstract qualifier.",
+                        NewInstance.classname, namespace))
```

Comment:
Same comment o the message text as above.

Reply:
Agreed. Done.  Added CreateInstance failed as lead-in